src/models/submission/mod.rs

```diff
@@ -138,5 +139,37 @@ pub struct SubmissionData {
     pub num_reports: Option<u64>,
 }
 
+/// SubmissionDataPreview
+#[derive(Debug, Serialize, Deserialize)]
+pub struct SubmissionDataPreview {
+    /// List of preview images.
+    pub images: Vec<SubmissionDataPreviewImage>,
+    /// This is `true` if the preview is enabled.
+    pub enabled: bool,
+}
+
+/// SubmissionDataPreviewImage
+#[derive(Debug, Serialize, Deserialize)]
+pub struct SubmissionDataPreviewImage {
+    /// Object for the main preview image containing URL, width and height.
+    source: SubmissionDataPreviewImageSource,
+    /// List of objects describing all available resolutions of the preview image.
+    resolutions: Vec<SubmissionDataPreviewImageSource>,
+    // TODO: skipped variants
```

Comment:
For all examples I've seen this has been just an empty object (`{}`) so I skipped it